src/site/stages/build/process-cms-exports/schemas/transformed/taxonomy_term-health_care_service_taxonomy.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['taxonomy_term-health_care_service_taxonomy'] },
+    entity: {
+      type: 'object',
+      properties: {
+        entityType: { enum: ['taxonomy_term'] },
+        entityBundle: { enum: ['health_care_service_taxonomy'] },
+        name: { type: 'string' },
+        description: {
+          type: 'object',
+          properties: {
+            processed: {
+              type: ['string', 'null'],
+            },
+          },
+          required: ['processed'],
+        },
+        parent: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              entity: {
+                type: 'object',
+                properties: {
+                  name: { type: 'string' },
+                },
+                required: ['name'],
+              },
+            },
+            required: ['entity'],
+          },
+          maxItems: 1,
```

Comment:
Do we need to make this an array, then? Is it expected to be an array in the templates?

Reply:
Yeah - it looks like the templates are [expecting an array of one item](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/layouts/health_care_region_health_services_page.drupal.liquid#L37).

And here is a snippet from `pages.json` as well:

![image](https://user-images.githubusercontent.com/2008881/71844440-f99a3e00-307a-11ea-8adb-454ded860e51.png)